docs/community/exhibition-checklist.md

```diff
@@ -0,0 +1,116 @@
+Be kind to your fellow volunteers and try to be flexible so everyone has a chance to volunteer at the booth, but also experience the event as an attendee too.
+
+
+## Preparation checklist
+
+Sections below are in sequential order.
+
+### Publish call for projects on FOSS@MAGIC website
+
+* **What**: Make a new announcement on FOSS@MAGIC website to ask community members to submit projects for exhibition (or get help on starting projects)
+* **Who**: FOSS@MAGIC staff, or student volunteers asked by FOSS@MAGIC staff
+* **When**: About 4 months before event
+* **Where**: [FOSS@MAGIC website - Announcements](https://fossrit.github.io/announcements/)
+* **Why**: Build community interest for upcoming event and support prospective exhibitors in their project work during the semester
+* **How**: See [Website - How to create announcements](/infra/website "Scroll down to 'How to create announcements' section")
+
+### Submit event registration
+
+* **What**: Submit application form to attend an event/conference as an exhibitioner
+* **Who**: FOSS@MAGIC staff, or student volunteers asked by FOSS@MAGIC staff
+* **When**: Typically 3-4 months _before_ event (check event website for deadlines)
+* **Where**: Event/conference website
+* **Why**: In order to get a booth to exhibit, we need to apply and pay any fees, if required
+* **How**:
+    * Events we frequently attend have copy+paste text available in FOSS@MAGIC Google Drive
+    * However, most info we submit is also found on our [public website](https://fossrit.github.io) (see [_About_](https://fossrit.github.io/about/) and [_Projects_](https://fossrit.github.io/projects/))
+
+### Add exhibited projects to FOSS@MAGIC website
+
+Any project we exhibit at an event needs a project profile on the FOSS@MAGIC website.
+See [Website - How to add projects](/infra/website "Scroll down to 'How to add projects' section") for how to do this.
+
+### Add event to FOSS@MAGIC website
+
+Any event, conference, or show we attend needs an event profile on the FOSS@MAGIC website.
+The website does not support recurring events, so each individual event should get its own profile.
+See [Website - How to create events](/infra/website "Scroll down to 'How to create events' section") for how to do this.
+
+### Pack inventory for exhibition booth
+
+* **What**: Pack supplies, hardware, and other inventory for an exhibition booth
+* **Who**: Booth volunteers
+* **When**: 1-2 days before event travel
+* **Where**: In RIT MAGIC Suite / Prof. Stephen Jacob's office
+* **Why**: To prepare for the exhibit and **test all hardware BEFORE event to avoid on-site problems on event day**
+* **How**: Assemble the following items:
+    * Tablecloth (x1)
+    * Printed brochures/handouts (as available, [ask FOSS@MAGIC contacts](/about/contact) for guidance)
+    * Laptops (x2)
+    * Monitors (x2)
+    * USB keyboard (x1)
+    * USB mouse (x1)
```

Comment:
We've presented/tabled at a number of conferences without these for what it's worth.

Reply:
This is probably more relevant for Maker Faire / Imagine than actual industry conferences. I will add a note to the _Inventory tips_ section for events with a longer travel haul involved. Thanks for pointing it out.